src/parse_args.ts

```diff
@@ -65,7 +65,14 @@ program
     '--dry-run',
     "don't actually execute anything, report only registered journeys"
   )
-  .option('--journey-name <name>', 'only run the journey with the given name')
+  .option(
+    '--match <name>',
+    'run only journeys with a name that matches the glob'
+  )
+  .option(
+    '--tags <name...>',
```

Comment:
Since we're using globs do we support negation too?

Reply:
yes it should, Let me add a test here.